### ISSUE TITLE
[com_banners] Undesired margin bottom and green background with calendar field

### ISSUE THREAD

**Issue by Quy** [joomla/joomla-cms#15269]
### Steps to reproduce the issue
Go to Components > Banners > Tracks
Click Search Tools button
See screenshot

![banners-tracks](https://cloud.githubusercontent.com/assets/368084/24979583/a2d10e96-1f89-11e7-85c7-134b6fc5e497.jpg)

**Comment by franz-wohlkoenig** [comment]
Issue confirmed.
### System information
3.7.0-rc2 (nightly Build)
macOS Sierra, 10.12.4
Firefox 52 (64-bit)
#### MAMP 4.1.1
- PHP 7.0.15
- MySQLi 5.6.35

**Comment by rjcf18** [comment]
I confirm this issue.

![deepinscreenshot20170414014721](https://cloud.githubusercontent.com/assets/6710380/25029361/78114fbe-20b4-11e7-9ae0-836d6d98c350.png)


**System information (as much as possible)**

Joomla! 3.7.0-rc2
OS: Linux Deepin 15.3
Kernel: 4.4.0-3-deepin-amd64
Google Chrome: 56.0.2924.76 (64 bit)
PHP: 7.0.12-2

**Comment by dgrammatiko** [comment]
Is this still valid in 3.7.4?

**Comment by wojsmol** [comment]
@dgt41 Grean backgroud confirmed on 3.7.4.

**Comment by dgrammatiko** [comment]
@ciar4n @C-Lodder can you take a look at this one? It looks like css related

**Comment by wojsmol** [comment]
background is defined in `.input-append .active, .input-prepend .active` selector.

**Comment by dgrammatiko** [comment]
@wojsmol I think the conflict comes from the search tools...

**Comment by C-Lodder** [comment]
The issue simply comes from:

```
.input-append, .input-prepend {
    margin-bottom: 9px;
}
```
which is Bootstrap's CSS.

So the following LESS required to fix would be:

```
.js-stools-field-filter {
    .input-prepend,
    .input-append {
        margin-bottom: 0;
    }
}
```

**Comment by dgrammatiko** [comment]
@C-Lodder i guess there should be a `background-color` reset for the green background or not?

**Comment by C-Lodder** [comment]
I have absolutely no idea what validation classes are being used in J3, so not sure if the green BG should be there or not.

**Comment by ciar4n** [comment]
Is there a particular reason why the active class is been added to these fields?

**Comment by brianteeman** [comment]
@ciar4n not that I can see but I also cannot see where it is coming from

**Comment by ghost** [comment]
closed as having Pull Request #18231

**Comment by C-Lodder** [comment]
@franz-wohlkoenig That PR adresses the colour, not the unecessary bottom margin. So this needs to be re-opened

**Comment by ghost** [comment]
reopen as stated above.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15269">issues.joomla.org/tracker/joomla-cms/15269</a>.</sub>

**Comment by brianteeman** [comment]
I added the code for the margin to the PR at #18231

**Comment by ghost** [comment]
closed as having Pull Request #18231
